validator/client/validator.go

```diff
@@ -26,7 +26,10 @@ type validator struct {
 	pubKey          []byte
 }
 
-// Initialize
+// Initialize sets up a validator client's secrets and generates the necessary deposit data
```

Comment:
We actually might not even need this anymore since the genesis and slot ticker are initialized later